.github/workflows/docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Docs
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
+      - name: Install dependencies
+        run: npm install
```

Comment:
I'd go with `npm ci` instead of `npm install` here, to ensure you're using the lockfile: https://docs.npmjs.com/cli/v7/commands/npm-ci

Reply:
Yes indeed, it is better with `npm ci`. I will make another PR to take into account your comment.